packages/angular/src/generators/scam-pipe/scam-pipe.ts

```diff
@@ -21,12 +23,14 @@ export async function scamPipeGenerator(tree: Tree, schema: Schema) {
     'pipe'
   );
   await angularPipeSchematic(tree, {
```

Comment:
I know it might seem redundant, but should we set standalone: false here?

In case Angular flips that to true?

Reply:
Yeah, good call, not an issue right now, but can be an issue in future versions. Will make that change in another PR where I make these changes for the SCAM directive generator.